examples/wallet_examples/get_balance/get_balance.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/example_setup"
+	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/show"
+)
+
+var (
+	// Limit is the number of outputs to retrieve per page for balance calculation.
+	Limit = uint32(100)
+
+	// Originator specifies the originator domain or FQDN used to identify the source of the balance request.
+	// NOTE: Replace "example.com" with the actual originator domain or FQDN in real usage.
+	Originator = "example.com"
+
+	// Basket is the basket to get balance from (holds automatically managed "change").
+	Basket = "default"
+)
+
+// This example demonstrates how to calculate the balance of a wallet by summing
+// all satoshis from outputs in the default basket using pagination.
+func main() {
+	show.ProcessStart("Get Wallet Balance")
+	ctx := context.Background()
+
+	alice := example_setup.CreateAlice()
+
+	aliceWallet, cleanup := alice.CreateWallet(ctx)
+	defer cleanup()
+
+	show.Step("Alice", "Calculating wallet balance")
+
+	var balance uint64
+	var offset uint32 = 0
```

Comment:
To keep things consistent you don't have to assign to zero to the empty `uint32`
```suggestion
	var offset uint32
```